nodejs/spa/src/main/webapp/index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Keycloak Single-Page Application Example</title>
+    <script type="text/javascript" src="KC_URL/js/keycloak.js"></script>
```

Comment:
The default type of a script in a HTML5 document is `text/javascript`, so this can be omitted.
```suggestion
    <script src="KC_URL/js/keycloak.js"></script>
```